src/server/__main__.py

```diff
@@ -57,7 +57,9 @@ async def legacy(request):
         return sanic.response.raw(b'', status=405)
     auth = request.headers.get('Authorization')
     result = legacy_service.call(request.body, {'auth': auth})
-    return sanic.response.text(result)
```

Comment:
What was wrong here?

Reply:
The sanic text() method sets the content-type to "text/plain". It worked, but the content-type should be "application/json" for picky clients.
The jsonrpc11base library is best used when it operates on strings (since that is what json is), that is, pure json, rather than Python native values. One reason is that JSON-RPC should be the one to catch parsing errors and return the appropriate error. Also, it is encapsulates the json concerns of JSON-RPC to jsonrpc11base internals - the methods supplied by the implementation just need to use json-compliant python-native values.

Reply:
👍